### ISSUE TITLE
Peerio relationship to miniLock

### ISSUE THREAD

**Issue by Brenkon** [PeerioTechnologies/peerio-client#105]
miniLock is a terrific application to encrypt files/folders locally - while peerio seems to encrypt to the cloud using the same or a highly similar encryption engine. 

Any consideration given to merge these 2 products into 1 -  so that peerio has capability for BOTH cloud and local encryption ?

**Comment by skylarnagao** [comment]
Peerio does utilize miniLock and does encrypt locally!

**Comment by skylarnagao** [comment]
@Brenkon Sorry, did you mean local encryption and saving files locally?

**Comment by Brenkon** [comment]
Yes! We misunderstood each other :) I get that peerio encrypts on my machine - that's why I am interested in it.

However, I was asking if peerio can (or will) be able to save encrypted files locally. If not, it means using miniLock for that use case, and peerio for cloud storage / sharing. I would prefer 1 app that handles both use cases.

Thanks!

**Comment by timdinger** [comment]
The suggested feature could also be useful for using a shared computer. 

Right now if someone sends me a sensitive file, Peerio is a great solution. If I want to keep it locally and keep it secure, I download the now unencrypted file and have to re-encrypt it with miniLock (or some other solution). It would be nice to be able to download the encrypted file and leave it encrypted until I needed it (or perhaps until I have a non-shared machine to use). 

And yes, I understand that this is a different application to the idea of file saving feature described above.
